ext-openais/3393.xml

```diff
@@ -181,7 +181,7 @@ ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEAL
 				<Operations>W</Operations>
 				<MultipleInstances>Single</MultipleInstances>
 				<Mandatory>Mandatory</Mandatory>
-				<Type></Type>
+				<Type>String</Type>
```

Comment:
This is Writable resource but description says "When executed, ..." so maybe this should be an executable resource? Same question applies to all other similar resources.

Reply:
Hi Mert, thx for your feedback! We will fix the issues you raised. Do you prefer a new merge request with a new branch (kuenzge-patch-2) or can we re-use the same branch?